kano/network.py

```diff
@@ -233,8 +233,11 @@ def getList(self, unsecure=False, first=False, debug=False):
         '''
 
         def sortNetworks(adict):
-            x, z = adict['quality'].split('/')
-            factor = int(x) / float(z)
+            if '/' in adict['quality']:
+                x, z = adict['quality'].split('/')
+                factor = int(x) / float(z)
+            else:
+                factor = int(adict['quality']) / 100
```

Comment:
Ah, I'm not familiar with the intricacies of float v integer operations in Python (old Perl hacker). Adding a decimal point is an elegant fix, but since I'm a noob I'll use the float() to make it more obvious. Thanks!